master/internal/rm/agentrm/agent_resource_manager.go

```diff
@@ -381,6 +382,9 @@ func (a *agentResourceManager) Receive(ctx *actor.Context) error {
 	case taskContainerDefaults:
 		ctx.Respond(a.getTaskContainerDefaults(msg))
 
+	case sproto.GetNonDaiJobs:
+		return rmerrors.ErrNotSupported
```

Comment:
confusingly, if this were what we wanted, you would want `ctx.Respond(err)`. `return err` will just crash the actor.